src/chat/Chat.js

```diff
@@ -12,6 +12,12 @@ import NoMessages from '../message/NoMessages';
 import ComposeBoxContainer from '../compose/ComposeBoxContainer';
 import UnreadNoticeContainer from './UnreadNoticeContainer';
 
+const stylesObj = StyleSheet.create({
```

Comment:
We usually call this `styles` or if it conflicts with the other property coming from the 'context' (which in this case it does) we call it `componentStyles`. The only reason to want to change this is consistency.